WooCommerce/WooCommerceUITests/Screens/Login/GetStartedScreen.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+import XCTest
+
+private struct ElementStringIDs {
+    static let navBar = "WordPress.GetStartedView"
+    static let emailTextField = "Email address"
+    static let continueButton = "Get Started Email Continue Button"
+}
+
+class GetStartedScreen: BaseScreen {
```

Comment:
nit: could this be `final`?

```suggestion
final class GetStartedScreen: BaseScreen {
```

Reply:
These "screens" are lifted from WordPress. I didn't want to change the conventions much, as those tests are mostly maintained by platform 9. 

That's why I didn't declare anything as final, and that's also why there is code that is actually not used at the moment, but will be used soon, as soon as we add more tests to cover the parts of the login flow that are not covered yet (in #3347 )

What if I declare all these classes as final and we merge, leaving the code that is unused now untouched?

Reply:
Gotcha, so these new `BaseScreen` subclasses are from WPiOS written/maintained by Platform9! No worries about the nit changes then, go for it :shipit: